.github-scheduled-workflows/publish-main.yaml

```diff
@@ -39,14 +27,21 @@ jobs:
         if: ${{ success() && inputs.prev_version }}
         uses: fjogeleit/yaml-update-action@main
         with:
-          valueFile: 'config/manifests/bases/*-operator.clusterserviceversion.yaml'
+          valueFile: 'config/manifests/bases/gingersnap-operator.clusterserviceversion.yaml'
           branch: main
           message: 'Next Version ${{ env.release-version }}'
           commitUserName: 'infinispan-qe-bot'
           commitUserEmail: 'q*@infinispan.org'
           token: ${{ secrets.PAT_TOKEN }}   
           changes: |
             {
-              "replaces": "gingersnap.v${{ env.replace_version }}"
+              "spec.replaces": "gingersnap.v${{ env.replace_version }}"
             }
+
+      - name: Remove scheduled job after 10 attempts
+        uses: cardinalby/unschedule-job-action@v1
+        if: github.run_number > 10
```

Comment:
We need to declare two `unschedule-job-action` steps:

1. To limit the number of executions with `if: github...`. This should be the first step in the job, see this [example](https://github.com/cardinalby/schedule-job-action#limit-attempts-number) and explanation.
2. To unschedule the job on successful completion. This should not have a `if` statement as it should only be executed if all other steps complete successfully.